proposals/stage-1/PROJECT_ONBOARDING/PROJECT_ONBOARDING.md

```diff
@@ -0,0 +1,22 @@
+# Onboarding Checklist
+
+This is an informational checklist to help projects onboard into the OpenJS Foundation. If you have any questions or need help, the OpenJS Foundation CPC is available to assist.
+
+- [ ] Adopt the OpenJS Foundation Code of Conduct
+- [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
+- [ ] Transfer official domains to OpenJS Foundation
+- [ ] Identify and document other core project infrastructure
+- [ ] Choose either CLA or DCO; implement appropriate tool
```

Comment:
is a CLA really required? https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/ seems convincing to me

Reply:
No, not at all - projects can chose what works for them.

Reply:
Thanks, that makes sense. The wording here (and the "checklist" form) implied to me that all were required.

Reply:
Good feedback, I'll write some clarifying copy here!